cookiecutter.json

```diff
@@ -13,8 +13,7 @@
   ],
   "search": [
     "opensearch2",
-    "opensearch1",
-    "elasticsearch7"
+    "opensearch1"
```

Comment:
Per https://inveniordm.docs.cern.ch/releases/versions/version-v12.0.0/#breaking-changes I think opensearch1 support is also gone. And in that case can we remove the search option entirely?

Reply:
Yes, we can remove it as well, as long as the CERN team doesn't have a different opinion. Perhaps we could handle it in a separate PR.

Reply:
I removed it as a separate commit now.